### ISSUE TITLE
Create a story to teach basic CLI skills

### ISSUE THREAD

**Issue by njbennett** [pivotal/cf-onboarding#217]
The stories currently assume a level of command-line skill that's standard for 3rd month engineers. Docs writers, TPMs, PMs, and other roles may not had the opportunity to develop these skills. 

An "Intro to the CLI" story would help these folks learn the CLI techniques they need to complete later stories quickly.

I asked Jesse to take a shot at this.

**Comment by ljfranklin** [comment]
Yeah sounds great! Going to put an open for contribution label on this, happy to chat if you have questions.

**Comment by ameowlia** [comment]
@anEXPer - This came up in my onboarding week and I took a stab at it here: https://github.com/pivotal/cf-onboarding/pull/252
